app/core/services/stripe.coffee

```diff
@@ -1,7 +1,9 @@
 publishableKey = if application.isProduction() then 'pk_live_27jQZozjDGN1HSUTnSuM578g' else 'pk_test_zG5UwVu6Ww8YhtE9ZYh0JO6a'
 
 if me.isAnonymous()
-  module.exports = {}
+  module.exports = {
+    openAsync: _.noop
```

Comment:
Are we sure about this?  If we never call openAsync on a non logged in stripe user, why even have this?  Maybe it could trigger a login just incase?